packages/components/src/components/vertical-tab/vertical-tab-content-container.tsx

```diff
@@ -63,58 +84,32 @@ const Content = ({ is_routed, items, selected }) => {
                     })}
                 </Switch>
             ) : (
-                <TabContent key={selected_item.label} className='item-id' setSideNotes={addToNotesQueue} />
+                <TabContent key={selected_item!.label} className='item-id' setSideNotes={addToNotesQueue} />
             )}
             {selected.has_side_note && (
                 // for components that have side note, even if no note is passed currently,
                 // we want to keep the column space for side note
-                <SideNotes selected_item={selected_item} side_notes={side_notes} />
+                <SideNotes side_notes={side_notes} />
             )}
         </React.Fragment>
     );
 };
 
 const VerticalTabContentContainer = ({
-    action_bar,
-    action_bar_classname,
     className,
-    id,
     is_floating,
     is_routed,
     items,
     selected,
     tab_container_classname,
-}) => {
+}: TVerticalTabContentContainer) => {
     return (
         <div
             className={classNames('dc-vertical-tab__content', {
                 'dc-vertical-tab__content--floating': is_floating,
                 [`dc-vertical-tab__content--${className}`]: className,
             })}
         >
-            {!is_floating && action_bar && (
```

Comment:
deleting this as it isn’t used

Reply:
@yauheni-deriv why do you think it's not used? i can see the opposite from the code. is_floating is being passed into this component as well as action_bar. this is part of its functionality, no?

Reply:
checklist is provided in parent card for this sprint

Reply:
action_bar  is not passed anywhere, except of stories. So if it isn’t used I suggest delete this prop and remove it from strories as well

Reply:
action_bar is passed from stories so let it be, it's a feature🙂

Reply:
action_bar has been restored